e2e/cypress/integration/team_settings/invite_members_spec.js

```diff
@@ -175,7 +175,7 @@ describe('Invite Members', () => {
 
         // By default, member don't have "InviteGuest" permission
         // should go directly to "InviteMembers" modal
-        it('Invite Members to Team as Member - invitation sent', () => {
+        it('MM-T1328 Invite Members to Team as Member - invitation sent', () => {
```

Comment:
Was this a different test case? 1328 is already keyed above as 'Invite Members - Existing Member not on the team' in member_invitation_ui_spec.

Reply:
Reverted for now.